src/Common/src/System/Security/Cryptography/DSAOpenSsl.cs

```diff
@@ -180,8 +180,8 @@ protected override byte[] HashData(byte[] data, int offset, int count, HashAlgor
             protected override byte[] HashData(Stream data, HashAlgorithmName hashAlgorithm) =>
                 AsymmetricAlgorithmHelpers.HashData(data, hashAlgorithm);
 
-            protected override bool TryHashData(ReadOnlySpan<byte> source, Span<byte> destination, HashAlgorithmName hashAlgorithm, out int bytesWritten) =>
-                AsymmetricAlgorithmHelpers.TryHashData(source, destination, hashAlgorithm, out bytesWritten);
+            protected override bool TryHashData(ReadOnlySpan<byte> data, Span<byte> destination, HashAlgorithmName hashAlgorithm, out int bytesWritten) =>
```

Comment:
Is it strange to have "data" and "destination" rather than "source" and "destination"?  I'm ok with it if you are, just highlighting the discrepancy.

Reply:
It makes sense to me.  "Hash `data` into `destination`".

In the allocating versions the output data was a return object, so it didn't have a name.  So a common "destination" seems fine, and no more ambiguous than the allocating version.  But the input variable names were (in some cases) the only thing that told you the expected shape of the input data.

`output` is a fine alternative.  Or if we want to bother with contextual output names `destination` values could become `hash` and `signature`.... though for Encrypt and Decrypt we have "data" as input for both, so they'd be `(data, data)` (which is illegal) or `(dataIn, dataOut)`, or some variation thereon.

Reply:
I'm ok with it. Do we consistently use destination then as the output name? I thought I'd seen a new one that had signature as the output name?

Reply:
They seem to all be `Span<byte> destination`.  DSA has `VerifySignature(ReadOnlySpan<byte> hash, ReadOnlySpan<byte> signature)`, and RSA/ECDSA have `VerifyHash(ReadOnlySpan<byte> hash, ReadOnlySpan<byte> signature)`, but those are input signatures, not output signatures.

Reply:
Okey dokey